pkgs/top-level/all-packages.nix

```diff
@@ -9943,7 +9943,16 @@ let
 
   ekho = callPackage ../applications/audio/ekho { };
 
-  electrum = callPackage ../applications/misc/electrum { };
+  electrum = callPackage ../applications/misc/electrum {
```

Comment:
When I looked at other examples, it seemed most were passing stuff explicitly, but if it is acceptable to pass pythonPackages, then I strongly prefer doing so.